developerportal/templates/footer.html

```diff
@@ -1,26 +1,25 @@
 <footer class="mzp-c-footer">
   <div class="mzp-l-content">
-    <img class="mozilla-developer-logo" src="/static/img/logos/developer/white.svg" alt="Mozilla Developer logo" />
+    <a href="/"><img class="mozilla-developer-logo" src="/static/img/logos/developer/white.svg" alt="Mozilla Developer logo" /></a>
+    {% if ABOUT_LINK %}
+    <a class="footer-about-link" href="{{ABOUT_LINK}}">About this site</a>
+    {% endif %}
   </div>
 
-  <div class="social-links">
-    <section class="mzp-l-content">
-      <div class="mzp-c-footer-sections">
-        <section class="mzp-c-footer-section link">
-          <a href="https://hacks.mozilla.org/" rel="noopener"><img src="/static/img/logos/mdn-dino-white.svg" alt="Mozilla Hacks Blog logo" />Mozilla Hacks Blog</a>
-        </section>
-        <section class="mzp-c-footer-section link">
-          <a href="https://twitter.com/mozhacks" rel="noopener"><img src="/static/img/icons/social/twitter/white.svg" alt="Twitter logo" />@mozhacks</a>
-        </section>
-        <section class="mzp-c-footer-section link">
-          <a href="https://www.youtube.com/user/mozhacks" rel="noopener"><img src="/static/img/icons/social/youtube/white.svg" alt="YouTube logo for Mozilla Hacks" />@mozhacks</a>
-        </section>
-        <section class="mzp-c-footer-section link">
-          <a href="https://www.youtube.com/MozillaDeveloper" rel="noopener"><img src="/static/img/icons/social/youtube/white.svg" alt="YouTube logo for Mozilla Developer" />@MozillaDeveloper</a>
-        </section>
-      </div>
-    </section>
-  </div>
+  <ul class="mzp-l-content social-links mzp-c-footer-sections">
+    <li class="mzp-c-footer-section">
+      <a class="social-link mozilla" href="https://hacks.mozilla.org/" rel="noopener">Mozilla Hacks Blog</a>
+    </li>
+    <li class="mzp-c-footer-section">
+      <a class="social-link twitter" href="https://twitter.com/mozhacks" rel="noopener">@mozhacks</a>
+    </li>
+    <li class="mzp-c-footer-section">
+      <a class="social-link youtube" href="https://www.youtube.com/user/mozhacks" rel="noopener">@mozhacks</a>
+    </li>
```

Comment:
Because the text content for these two links is the same, users with screen readers will not know the difference between the two unless they know the URL. Testing with VoiceOver in Safari, only the text is readout.

I would, therefore, recommend that we give these ARIA labels to distinguish them from each other:

```suggestion
    <li class="mzp-c-footer-section">
      <a class="social-link twitter" href="https://twitter.com/mozhacks" rel="noopener" aria-label="Follow @mozhacks on Twitter">@mozhacks</a>
    </li>
    <li class="mzp-c-footer-section">
      <a class="social-link youtube" href="https://www.youtube.com/user/mozhacks" rel="noopener" aria-label="Visit @mozhacks on Youtube">@mozhacks</a>
    </li>
```